tests/test_loky_backend.py

```diff
@@ -546,6 +550,101 @@ def test_compatibility_openmp(self):
         p.join()
         assert p.exitcode == 0
 
+    @pytest.mark.skipif(sys.platform == "win32",
+                        reason="The loky process are only defined for posix.")
+    @pytest.mark.parametrize("run_file", [True, False])
+    def test_interactively_define_process_no_main(self, run_file):
+        code = '\n'.join([
+            'from loky.backend.process import PosixLokyProcess',
+            'p = PosixLokyProcess(target=id, args=(1,), ',
+            # when using -c option, we don't need the safeguard if __name__ ..
+            # and thus test LokyProcess without the extra argument. For running
+            # a script, it is necessary to use init_main_module=False.
+            '                     init_main_module=False)' if run_file else
+            '                     init_main_module=True)',
```

Comment:
Please use a formatted string:` "init_main_module={0}" ... .format(run_file)` instead of the ternary operator hack.